index.rst

```diff
@@ -13,17 +13,188 @@
+
+**After:** End this madness.
+
+**Discussion:**
+
+The need to maintain two different branches has been eliminated with mobu's ability to easily be configured to run off different branches for cases where it is useful to have an "in-development" version deployed.
+Hence cherry-picking is just annoying with no particular benefit.
+
+Outputs
+-------
+
+**Before:** Notebook contributors need to remember to clear outputs before checking in new versions of the notebooks.
+
+**After:** Have this happen automatically (via pre-commit hook or similar), or at least raise a CI warning if there is output checked in.
+
+**Discussion:**
+
+Having the human remember to clear outputs before saving and checking in is error prone. Even if the notebook ends in clear_outputs(), it still implies it was run to the end before commit.
+Ideally something like https://github.com/srstevenson/nb-clean would be integrated in the development workflow.
+
+This may also be of use to other notebook repo maintainers.
+
+
+Write-Only
+----------
+
+**Before:** Notebooks are checked out write-only in nublado to avoid conflicts
+
+**After:** Tutorial-notebooks continue to be write-only (this has saved a lot of support headaches) but system-test notebooks can be read-write to allow for easier guided troubleshooting.
+
+Directories
+-----------
+
+**Before:** Mobu runs notebooks at the root level of repos [?]
+
+**After:** Indicate to mobu whether to run a subdirectory (through eg. a .run file)
+
+**Discussion:** It would be nice to have a top-level directory structure for multiple runable repos, particularly for the data curation notebooks.
+This is not terribly important.
+
+Summit
+------
+
+**Before:** Mobu does not run at the summit
+
+**After:** There are system-test notebooks that probe basic "telescope" functionality (eg communication with DDS)
+
+**Discussion:** These had better be passive, we don't want to move the telescope or anything.... We should check what if any protections there are for this, eg is there further authorisation required to perform certain tasks
+
+Timing
+------
+
+**Before:** We have no timing information related to whole-notebook runs or per-cell runs
+
+**After:** Notebook and/or cell execution time can be monitored for duration
+
+**Discussion:** This has been controversial in discussion with the reasonable argument that notebook execution relies on too many factors and excursions from the norm are not determinative. The other side of the argument is that metrics would indicate the statistical as-is user experience for execution times and can provide at least coarse statistics (if not for alerting, at least for inspection).
+
+Any metrics should arguably be dispatched to sasquatch for monitor dogfooding purposes
+
+
+Recommended
+-----------
+
+**Before:** There is no automated checking of future recommended candidates
```

Comment:
This is not entirely true. We test latest weekly on data-int (I keep meaning to move that to data, but haven't yet).

Reply:
The problem is the latest weekly is not necessarily the future recommended candidate. For example, w47 was the recommended candidate even after w48 came out.

Reply:
Oh, yeah, I understand, I just wanted to note that every release candidate has in fact been tested with the tutorial notebooks by mobu during the time it passed through being latest weekly. Not quite the same thing, though, and I have no objections to setting this up.

Reply:
okay will clarify

Reply:
clarified https://github.com/lsst-sqre/sqr-080/commit/6962e2cf07c268f76c1ee04413c4e7cf436dfb4c